eng/placeholderpackaging.targets

```diff
@@ -0,0 +1,26 @@
+<Project>
+
+  <PropertyGroup>
+    <!-- Prevent copying the build DLL into the bin. -->
+    <SkipCopyBuildProduct>true</SkipCopyBuildProduct>
+    <!-- Prevent restoring packages from frameworks that should not be built. -->
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <!-- Prevent including the build output in the package. -->
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <!-- Hook up a target to include the placeholder file in the package. -->
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificBuildOutput);_AddPlaceholderFileToPackage</TargetsForTfmSpecificContentInPackage>
+  <!-- TFMs that are only used as placeholders should not include any default items. -->
```

Comment:
nit: indenting.
```suggestion
    <!-- TFMs that are only used as placeholders should not include any default items. -->
```

Reply:
Viktor left me extra feedback for a follow up PR [here](https://github.com/dotnet/maintenance-packages/pull/30#discussion_r1427332859). I'll group it with this change.